src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -1369,6 +1369,36 @@ namespace winrt::TerminalApp::implementation
         return _root ? _root->AlwaysOnTop() : false;
     }
 
+    void AppLogic::IdentifyWindow()
+    {
+        if (_root)
+        {
+            _root->IdentifyWindow();
+        }
+    }
+
+    winrt::hstring AppLogic::WindowName()
+    {
+        return _root ? _root->WindowName() : L"";

```

Comment:
```suggestion
        return _root ? _root->WindowName() : {};
```
I could be wrong, but I think we're supposed to return the null ctor instead of `L""` because something something hstrings?

Reply:
This will not compile. Ternary expressions are ... difficult

Reply:
```suggestion
        return _root ? _root->WindowName() : winrt::hstring{};
```